### ISSUE TITLE
Dimmer value is not updated when HomeKit is started

### ISSUE THREAD

**Issue by Skjaar** [nfarina/homebridge#2060]
Dunno if this is a bug or if its me using it wrong but here goes...

### Explanation
I'm writing a plugin that fetches a dimmer device value to pass it along to HomeBridge. When I start HomeKit the state of the device is not updated correctly. It is shown as "Off" even thou the dimmer device is "On" (50% power). The other binary devices have the correct state when I start HomeKit. 

Status for binary switches are true or false and I have tried to pass true as a value and a number (Example: 50) that represents the current level the dimmer is at but it doesn't work. 

Is it a Homebridge bug or is it me using it wrong?

### Versions

- **Homebridge**: `homebridge --version 0.4.45` 
- **Node**:       `node --version 11.1.0`
- **OS**:         Mac OS X

**Comment by NorthernMan54** [comment]
Can you share a link to your GitHub repo with your plugin?

On plugin startup you should update hombeidge with current device status using updateValue. This should solve the issue.

**Comment by Skjaar** [comment]
Hi, sorry that I forgot the most important part. The link. Here it is. 
https://bitbucket.org/Skjaar/homebridge-nexabridge/src/master/

**Comment by Skjaar** [comment]
I see... so the SwitchOnChar. is handling on and of behaviour and the SwitchBrightnessChar. handles the dimmer value when you are adjusting it. I have added what you said Frank but when I adjust the brightness I'm noticing that two calls are made to the dimmer. Both from the SwitchOn and SwitchBrightness functions. 

I have updated the code in my repo. I have not implemented the getSwitchBrightnessChar. function yet so for now it uses the getSwitchOnChar. In case you were wondering what's going on there. :)
